transport/ws/src/main/java/org/kaazing/gateway/transport/ws/AbstractWsBridgeSession.java

```diff
@@ -69,22 +70,22 @@
 
     protected ScheduledExecutorService scheduler;
     protected DefaultLoginResult loginResult;
-    protected final ActiveExtensions wsExtensions;
+    protected final ActiveWebSocketExtensions wsExtensions;
```

Comment:
Can we keep the simpler name (ActiveExtensions)? It's more readable because shorter, and since the package is transport.ws we already know it's WebSocket extensions we're talking about.